pkg/lib/bundlev2/generate.go

```diff
@@ -0,0 +1,144 @@
+package bundlev2
```

Comment:
naming nit: what about "synthesize"?

Reply:
we have synthesize.go which is to synthesize. The generate is to generate the CSV. Dont you think this is necessary to separate the two?

Reply:
omg, I just realized you were referring to the package name. I will include it locally and put it in next or when i am squashing it.